pi_top_usb_setup/pages/run_setup.py

```diff
@@ -118,12 +122,18 @@ def run_setup(self):
             # Update packages
             self._update_system()
 
-            # Configure device
+            # Configure device: process json file
             self._configure_device()
 
+            # Copy files over to the device
+            self._copy_files_to_device()
+
             # Finish onboarding if necessary
             self._complete_onboarding()
 
+            # Run scripts
+            self._run_scripts()
```

Comment:
let's move this before complete onboarding so that if it fails we haven't done that committal step